x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/export_rules/route.ts

```diff
@@ -33,6 +34,9 @@ export const exportRulesRoute = (
       path: `${DETECTION_ENGINE_RULES_URL}/_export`,
       options: {
         tags: ['access:securitySolution'],
+        timeout: {
+          idleSocket: RULE_MANAGEMENT_IMPORT_EXPORT_SOCKET_TIMEOUT_MS,
```

Comment:
Is the import timeout updated in a separate PR?

Reply:
Yes, it's been updated [here](https://github.com/elastic/kibana/pull/177159).

Reply:
Finally set import timeout here just in case the other PR doesn't make it to main.